messaginginapp/src/main/java/io/customer/messaginginapp/ModuleMessagingInApp.kt

```diff
@@ -1,70 +1,30 @@
 package io.customer.messaginginapp
 
 import android.app.Application
-import androidx.annotation.VisibleForTesting
 import io.customer.messaginginapp.di.gistProvider
-import io.customer.sdk.CustomerIOConfig
-import io.customer.sdk.android.CustomerIO
 import io.customer.sdk.communication.Event
 import io.customer.sdk.communication.subscribe
 import io.customer.sdk.core.di.SDKComponent
 import io.customer.sdk.core.module.CustomerIOModule
 import io.customer.sdk.data.request.MetricEvent
-import io.customer.sdk.di.CustomerIOComponent
 
-class ModuleMessagingInApp
-@VisibleForTesting
-internal constructor(
-    override val moduleConfig: MessagingInAppModuleConfig = MessagingInAppModuleConfig.default(),
-    private val overrideDiGraph: CustomerIOComponent?
+class ModuleMessagingInApp(
+    config: MessagingInAppModuleConfig
 ) : CustomerIOModule<MessagingInAppModuleConfig> {
-    @JvmOverloads
-    @Deprecated(
-        "organizationId no longer required and will be removed in future",
-        replaceWith = ReplaceWith("constructor(config: MessagingInAppModuleConfig)")
-    )
-    constructor(
-        organizationId: String,
-        config: MessagingInAppModuleConfig = MessagingInAppModuleConfig.default()
-    ) : this(
-        moduleConfig = config,
-        overrideDiGraph = null
-    )
-
-    @JvmOverloads
-    constructor(
-        config: MessagingInAppModuleConfig = MessagingInAppModuleConfig.default()
-    ) : this(
-        moduleConfig = config,
-        overrideDiGraph = null
-    )
```

Comment:
Removed unused and deprecated constructors

Reply:
Lets make a follow up ticket to remove from docs, if they are still there?

Reply:
Sure, will do 👍🏻